projects/core/src/cart/event/cart.events.ts

```diff
@@ -26,11 +26,11 @@ export class CartAddEntrySuccessEvent extends CartEvent {
    * Event's type
    */
   static readonly type = 'CartAddEntrySuccessEvent';
-  productCode: string;
-  quantity: number;
-  entry: OrderEntry;
-  quantityAdded: number;
-  deliveryModeChanged: boolean;
+  productCode?: string;
+  quantity?: number;
+  entry?: OrderEntry;
+  quantityAdded?: number;
+  deliveryModeChanged?: boolean;
```

Comment:
For future reference:
We noticed that the `CartModification` model coming from backend can have all those properties present or absent, depending on the `?fields` in the cart occ endpoint configuration. None of those fields is mandatory for Spartacus to work (because we don't take those properties in account, we're just reloading whole cart afterwards).
That's why we cannot guarantee the presence of above properties in the event's payload. So we're relaxing the typing to optional.